chord_metadata_service/discovery/pydantic_models.py

```diff
@@ -67,24 +70,50 @@ class BaseMatchModel(BaseModel, abc.ABC):
     dataset: str | None = Field(default=None, title="Dataset ID")
 
 
+class ExperimentResultIndex(BaseModel):
+    url: str
+    format: Literal[
+        "BAI",  # BAM index files ( http://samtools.github.io/hts-specs/SAMv1.pdf "BAI" )
+        "BGZF",  # BGZip index files (often .gzi)
+        "CRAI",  # CRAM index files ( https://samtools.github.io/hts-specs/CRAMv3.pdf "CRAM index" )
+        "CSI",  # See http://samtools.github.io/hts-specs/CSIv1.pdf
+        "TABIX",  # See https://samtools.github.io/hts-specs/tabix.pdf
+        "TRIBBLE",
+    ]
+
+
+class ExperimentResultIndices(RootModel):
+    root: list[ExperimentResultIndex]
+
+
+# TODO: just merge this with the main serializer; there's no point in returning a subset -- possibly via drf-pydantic
 class MatchExperimentResult(BaseMatchModel):
     id: int = Field(..., title="Experiment result ID")
-    # TODO: experiments backlink
+    identifier: str = Field(..., title="Experiment result laboratory identifier")
+    description: str = Field(..., title="Description")
     filename: str | None = Field(..., title="File name")
     url: str | None = Field(..., title="URL")
-    # list of experiment_result_file_index objects (see experiments/schemas.py)
-    indices: list[dict] = Field(..., title="Indices")
-    file_format: str | None = Field(..., title="File format")
-    assembly_id: str | None = Field(..., title="Genome assembly ID")
+    indices: ExperimentResultIndices = Field(..., title="Indices")
+    file_format: ExperimentResultFileFormat | None = Field(..., title="File format")
+    data_output_type: Literal["Raw data", "Derived data"] | None = Field(..., title="Data output type")
+    usage: str | None = Field(..., title="Usage")
+    creation_date: str | None = Field(..., title="Creation date")
+    created_by: str | None = Field(..., title="Created by")
+    genome_assembly_id: str | None = Field(..., title="Genome assembly ID")
+    extra_properties: dict = Field(..., title="Extra properties")
+    # TODO: experiments backlink
 
 
 class MatchExperiment(BaseMatchModel):
     """
     Compact representation of an experiment for returning/rendering search responses.
     """
     id: str = Field(..., title="Experiment ID")
-    study_type: str
+    experiment_type: str = Field(..., title="Experiment Type")
+    study_type: str = Field(..., title="Study Type")
     results: list[MatchExperimentResult]
+    # backlink:
+    phenopacket: str | None = Field(..., title="Phenopacket ID")
```

Comment:
For the fields that have `None` as a possible type are they supposed to be required as indicated by the `...` in the Field object